src/test/clojure/clara/durability_rules.clj

```diff
@@ -29,7 +29,10 @@
 
 (defrule find-wind-speeds-without-temp
   "Rule using NegationNode"
-  [?w <- WindSpeed (= ?loc location)]
+  [?w <- WindSpeed
+   ;; Empty constraint to test the serializing EmptyList, see Issue 352 for more information
+   ()
```

Comment:
I'm not sure that its right to accept this, however removing it as a valid constraint would probably be seen as non-passive

Reply:
I think it is a bit odd to have the empty constraint along with something else. However, I agree that it doesn't really make sense to make it start failing now. It seems like something that could be done by people in practice fairly easily or maybe even intentionally for some visual preference.

Also, the empty list is "truthy" in Clojure, so I'd expect it to be evaluated that way.

Reply:
Yes this is an odd scenario, however i wanted to minimize the change to the rules that we use to test serialization, because one small change can vastly change the tests